.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,54 @@
+name: "CodeQL"
+
+on:
```

Comment:
If possible, please add a comment with a URL for the documentation of this configuration file. After that, I would remove most of the "default" comments left below. I do not insist on this.